archinstall/lib/packages/packages.py

```diff
@@ -1,7 +1,9 @@
 import ssl
 import urllib.request
 import json
-from typing import Dict, Any, Tuple, Iterator
+from typing import Dict, Any, Tuple, Iterator, List
+
+import archinstall
```

Comment:
Your IDE is acting up again ^^

Reply:
Yes it is :(

Reply:
Yeah no I get that, however, I can't see this import in my local branch nor can I see it in the `Files changes` tab either. Might this be from an older commit?

Reply:
Humm, maybe. I can see it in the change tab either.. GitHub playing tricks on my mind hehe. I’ll take a look tomorrow with less tired eyes and have a merge-spree ^^